### ISSUE TITLE
Arrow for OverlayPanel

### ISSUE THREAD

**Issue by Tuizi** [primefaces/primeng#1474]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => Search github for a similar issue or PR before submitting
[x] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**What is the motivation / use case for changing the behavior?**

The tooltip has the class `ui-tooltip-arrow` and this little arrow is always well aligned with the targeted input. I would like to know if this same arrow could be applied to the component **OverlayPanel**, to have an arrow targeting the button that trigger the overlay panel.

**Comment by alexandrevtukh** [comment]
+1 It would be helpful to have setting for arrow.

**Comment by cagataycivici** [comment]
Sounds good, we'll consider this.

**Comment by dhniels** [comment]
+1 on this. ng-bootstraps popover is really nice this way. being able to add an arrow would be great!

**Comment by cagataycivici** [comment]
Will do for 4.1.1

**Comment by dhniels** [comment]
Awesome! I've been working around it with an ::after pseudo element that forms a triangle using borders, then positioning it, but a more elegant and built in solution would be ideal!

**Comment by rogeronline** [comment]
any updates for this? @cagataycivici

**Comment by cagataycivici** [comment]
Implemented for 7.0.0-RC1

**Comment by rajjaiswalsaumya** [comment]
Any update on this ?
